bson-scala/build.gradle

```diff
@@ -18,17 +18,16 @@ description = "A Scala wrapper / extension to the bson library"
 archivesBaseName = 'mongo-scala-bson'
 
 dependencies {
-    compile project(path: ':bson', configuration: 'default')
+    implementation project(path: ':bson', configuration: 'default')
 }
 
 sourceSets {
     main {
         scala {
-            srcDirs += ["src/main/scala"]
```

Comment:
Did something stop working here?   It looks the same to me.

Reply:
It didn't like the `srcDirs +=` - so now we just set it once.